Testing/UnitTests/xASL_ut_function_xASL_adm_CheckFileCount.m

```diff
@@ -30,6 +30,12 @@
 fid2 = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_2.txt'), 'wt');
 fid3 = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_3.txt'), 'wt');
 
+try
+    fclose('all');
+catch
```

Comment:
also print message if you catch error

Reply:
I just removed it, this shouldn't fail anyway. Fix in bfbadb7